data/campaigns/Heir_To_The_Throne/scenarios/06_The_Siege_of_Elensefar.cfg

```diff
@@ -232,325 +232,381 @@
         [/message]
     [/event]
 
+
+
+# Thieves’ Guild
+# Handle the following cases:
+#  1) Enemies defeated before turn 4, four thieves join the revelry in town
+#  2) Occupy a village on the isle of Elensefar before turn 4, four thieves join Konrad's forces on turn 4
+#  3) Otherwise, on turn 4 the thieves appear and offer help
+#     a) Choose assistance infiltrating
+#        i) On turn 6, three thieves join Konrad's forces and show the ford.
+#        ii) On enemies defeated before turn 6, four thieves join the revelry in town
+#     b) Choose joining forces
+#        i) Occupy a village on the island, four thieves join Konrad's forces.
+#        ii) Never occupy a village, four thieves join the revelry in town.
+# Admitedly, (1) and (3a ii) are impossible without cheating, but they complete logic chart.
+# GL-2016JUL
+
+#define REGLOK XY
+    [kill]
+        id=Reglok
+    [/kill]
+    [unit]
+        id=Reglok
+        name= _ "Reglok"
+        type=Rogue
+        side=1
+        x,y={XY}
+        gender=male
+        [modifications]
+            {TRAIT_LOYAL}
+            {TRAIT_INTELLIGENT}
+        [/modifications]
+        {IS_LOYAL}
+    [/unit]
+#enddef
```

Comment:
Really curious why you're using [kill] and [unit] here instead of store and unstore.